dashboard/app/controllers/musiclab_controller.rb

```diff
@@ -1,5 +1,14 @@
 class MusiclabController < ApplicationController
   ANALYTICS_KEY = CDO.amplitude_api_key
+  NUM_MINI_PLAYER_PROJECTS = 5
+  # Hard-coded list of channel IDs for the mini player if DCDO flag is set to `true`
```

Comment:
We might want to specify which DCDO flag.  And assuming it's `"get_channel_ids_from_featured_projects_gallery"`, this list is actually used when it's set to `false` or not set at all?